fastlane_core/lib/fastlane_core/project.rb

```diff
@@ -172,6 +172,72 @@ def app_name
       return "App" # default value
     end
 
+    def build_for?
```

Comment:
Where is this getting used? I couldn't find a usage of this in https://github.com/fastlane/fastlane/pull/7091/files either?